04/User.js

```diff
@@ -1,3 +1,46 @@
 export default class User {
+  constructor(userData) {
+    const {email, password} = userData;
+    this.email = email;
+    this.password = password;
+    this.getEmail();
+    this.getPassword();
+    this.login();
+  }
+  getEmail() {
+    const isCorrectEmail = this.validateEmail();
 
-}
\ No newline at end of file
+    if (isCorrectEmail) {
+      return this.email;
+    } else {
+      throw new Error('Niepoprawny email');
+    }
+  }
+  getPassword() {
```

Comment:
Raczej funkcja z `get...` ma pobierać dane i nie walidować ich (to robimy zazwyczaj przy ustawianiu danych czyli np. w `constructorze`)